src/classes/BGE_ConfigurationWizard_CTRL.cls

```diff
@@ -58,41 +58,56 @@ public with sharing class BGE_ConfigurationWizard_CTRL {
                 try {
                     String descriptionField = (sObjectName == 'DataImportBatch__c') ? 'Batch_Description__c' : 'Description__c';
 
-                    String soql = 'SELECT Name, Id, ' +
-                            descriptionField + ', ' +
-                            'RequireTotalMatch__c, ' +
-                            'Active_Fields__c, ' +
-                            'Batch_Process_Size__c, ' +
-                            'Run_Opportunity_Rollups_while_Processing__c, ' +
-                            'Contact_Matching_Rule__c, ' +
-                            'Donation_Matching_Behavior__c, ' +
-                            'Donation_Matching_Rule__c, ' +
-                            'Donation_Date_Range__c, ' +
-                            'Process_Using_Scheduled_Job__c' +
+                    List<String> fields = new List<String>{
```

Comment:
You could use a set here to avoid duplicates in the future. #takeitorleaveit

Reply:
I'll take it!